### PR TITLE
[RLlib] Crr use workers for sampling and ray datasets

### PR DESCRIPTION
**Pull request by avnishn**
CRR currently doesn't sample directly from input reader and this pr fixes that.

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: rllib/algorithms/crr/crr.py [hunk 1]
```diff
@@ -60,7 +58,9 @@ def __init__(self, algo_class=None):
         self.tau = 5e-3
 
         # overriding the trainer config default
-        self.num_workers = 0  # offline RL does not need rollout workers
+        self.num_workers = 4  # offline RL does not need rollout workers
+        self.offline_sampling = True
+        self.min_iter_time_s = 10.0
```
Comment (sven1977) [inline rllib/algorithms/crr/crr.py:63]: Rename to `min_time_s_per_iteration`.

Comment (sven1977) [reply rllib/algorithms/crr/crr.py:63]: `offline_sampling` is not a default AlgorithmConfig parameter. We need to add it to the `training` method, correct?

File: rllib/tuned_examples/crr/pendulum-v1-crr.yaml [hunk 2]
```diff
@@ -5,9 +5,12 @@ pendulum_crr:
       evaluation/episode_reward_mean: -200
       training_iteration: 500
     config:
-        input:
-          - 'tests/data/pendulum/pendulum_replay_v1.1.0.zip'
+        input: "dataset"
```
Comment (sven1977) [inline rllib/tuned_examples/crr/pendulum-v1-crr.yaml:8]: Cool!

File: rllib/BUILD [hunk 3]
```diff
@@ -253,47 +253,47 @@ py_test(
...
-    size = "large",
-    srcs = ["tests/run_regression_tests.py"],
-    # Include an offline json data file as well.
-    data = [
-        "tuned_examples/crr/pendulum-v1-crr.yaml",
-        "tests/data/pendulum/pendulum_replay_v1.1.0.zip",
-    ],
-    args = ["--yaml-dir=tuned_examples/crr"]
-)
+# py_test(
```
Comment (sven1977) [inline rllib/BUILD:256]: Any reason, we need to disable these? :(

**Comment by avnishn** [comment]
@sven1977 I disabled the tests because they won't be able to run properly within the 300s timelimit with our dataset. 

This is also pretty much true for cql

File: rllib/BUILD (hunk 3 above)
Comment (avnishn) [reply rllib/BUILD:256]: see my comments in the comment section (not inline)

**Pull request closed without merging**